lir/transformers.py

```diff
@@ -89,35 +89,32 @@ class PercentileRankTransformer(sklearn.base.TransformerMixin):
 
     Fit:
     Expects:
-        - X is of shape (n, f) with n = number of measurements,
-        f = number of features
+        - `X` is a numpy array with one row per measurement
```

Comment:
this has made it less clear. Why not
```suggestion
        - `X` is a numpy array with one row per measurement, eg dimension (n, f) or (n, f, 2) 
        with n = number of measurements, f = number of features. 
        
```